Integrations/integration-EWSMailSender.yml

```diff
@@ -197,17 +210,36 @@ script:
 
 
     def test_module():
+        BaseProtocol.TIMEOUT = 15
         account = get_account(ACCOUNT_EMAIL)
-        account.root.tree()
         demisto.results('ok')
 
     config = prepare()
     args = prepare_args(demisto.args())
 
-    if demisto.command() == 'test-module':
-        test_module()
-    elif demisto.command() == 'send-mail':
-        demisto.results(send_email(**args))
+    try:
+        if demisto.command() == 'test-module':
+            test_module()
```

Comment:
Do you want to actually send an email in test? 
Will make the test more reliable

Reply:
Do you mean send a mail to the same mailbox?
don't know.. I think it's weird to send mail when test